README.md

```diff
@@ -92,25 +89,25 @@ By default VERSION is blank if you do not use the creator-platform-default.confi
 - pistachio_marduk_cc2520.dtb (for marduk_cc2520 board) or
 - pistachio_marduk_ca8210.dtb (for marduk_ca8210 board)
 
-For simplicity, let's assume that marduk_cc2520 PROFILE has been selected and VERSION as 1.0.0 hence the filenames will be as follows: 
-- openwrt-1.0.0-pistachio-pistachio_marduk_cc2520-uImage
-- openwrt-1.0.0-pistachio-pistachio_marduk_cc2520-uImage-initramfs
-- openwrt-1.0.0-pistachio-marduk-marduk_cc2520-rootfs.tar.gz
-- pistachio_marduk_cc2520.dtb
+For simplicity, let's assume that marduk_ca8210 PROFILE has been selected and VERSION as 1.0.0 hence the filenames will be as follows: 
+- openwrt-1.0.0-pistachio-pistachio_marduk_ca8210-uImage
+- openwrt-1.0.0-pistachio-pistachio_marduk_ca8210-uImage-initramfs
+- openwrt-1.0.0-pistachio-marduk-marduk_ca8210-rootfs.tar.gz
+- pistachio_marduk_ca8210.dtb
 
 ## Customising your OpenWrt
 You can configure OpenWrt from scratch but it's best to start from a base profile
 e.g. the one for the IMG pistachio board as it has some useful defaults.
 
 Simply select the package you need and add into the marduk profile:
 
-    $ vi target/linux/pistachio/marduk/profiles/marduk_cc2520.mk
+    $ vi target/linux/pistachio/marduk/profiles/marduk_ca8210.mk
 
 If you want to add new package, then simply add package name in the list of packages:
 
     define Profile/marduk
-    NAME:=Basic platform profile for Marduk with TI cc2520
-    PACKAGES:=kmod-i2c kmod-marduk-cc2520 kmod-sound-pistachio-soc \
+    NAME:=Basic platform profile for Marduk with Cascoda ca8210
+    PACKAGES:=kmod-i2c kmod-marduk-ca8210 kmod-sound-pistachio-soc \
```

Comment:
It's kmod-ca8210. Just copy paste from actual marduk_ca8210.mk to reflect latest changes
(If any)